Trappist/src/Promact.Trappist.Core/Controllers/TestController.cs

```diff
@@ -0,0 +1,39 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Promact.Trappist.DomainModel.Models.Test;
+using Promact.Trappist.Repository.Tests;
+using Promact.Trappist.Utility.Constants;
+
+namespace Promact.Trappist.Core.Controllers
+{
+    [Produces("application/json")]
+    [Route("api/tests")]
+    public class TestsController : Controller
+    {
+        private ITestsRepository _test;
```

Comment:
this name should be _testRepository. As you are creating an instance if Repository